app/models/qdm/tacoma/individual_result.rb

```diff
@@ -0,0 +1,33 @@
+module QDM
+  # IndividualResult stores the patient-level (population/clause) results for a patient/measure combination
+  class IndividualResult
```

Comment:
Is this Result object storing information for one population of a measure and there would be many results for many populations?

Reply:
This Result object is storing information for an entire measure, but only one patient. My intention with the referenced comment was to note that the clause-level results (e.g. whether the patient satisfies a clause) is included in this Result object.

Reply:
@okeefm please follow-up w @jbradl11 on this. doesn't seem like it's structured to handle multiple pops/strats for a single patient for an entire measure. I'm probably missing the conclusion of this conversation as it's from a while ago.